### DISCUSSION TITLE
Multiple API keys

### DISCUSSION THREAD

**Discussion by pke** [forwardemail/forwardemail.net#249] [Ideas]
It would be nice, if we could have multiple, revocable API keys.

**Comment by shaunwarman** [comment]
Just for curiosity in case other features are implied here. Could you expand on the use case? For example, is this to separate alias usage (and monitoring additions are in mind?) or just a way to safely add your own rotation mechanism?

**Reply by pke** [reply]
The second option. I would like to have different apps using different keys that I can easily revoke.
